test/redeyed-smoke.js

```diff
@@ -21,7 +21,7 @@ test('tap', function (t) {
     , 'resolve/test/precedence/'
     , 'is-relative/index.js'
     , 'is-buffer/test/'
-    , 'lodash/utility/'
+    , 'lodash/'
```

Comment:
So the smoke tests are flawed then :(
Particularly [here[(https://github.com/thlorenz/redeyed/blob/master/test/redeyed-smoke.js#L48-49)

We could improve on this a bit by having it ignore any lines that match `/^\s*(\/?*|\/\/)/`.
That should fix 90% of these don't you think?

If that works we can unignore a bunch of the ones we ignore now.